cloud/amazon/ec2.py

```diff
@@ -229,17 +229,17 @@
   exact_count:
     version_added: "1.5"
     description:
-      - An integer value which indicates how many instances that match the 'count_tag' parameter should be running. Instances are either created or terminated based on this value.
+      - An integer value which indicates how many instances that match the 'count_tags' parameter should be running. Instances are either created or terminated based on this value.
     required: false
     default: null
     aliases: []
-  count_tag:
+  count_tags:
     version_added: "1.5"
     description:
       - Used with 'exact_count' to determine how many nodes based on a specific tag criteria should be running.  This can be expressed in multiple ways and is shown in the EXAMPLES section.  For instance, one can request 25 servers that are tagged with "class=webserver".
     required: false
     default: null
-    aliases: []
+    aliases: ['count_tags']
```

Comment:
Should the alias here perhaps be `count_tag` (i.e. singular) like the original arg?